.github/workflows/check.yml

```diff
@@ -114,6 +115,9 @@ jobs:
       - name: Run tests and determine coverage
         run: cargo +${{ matrix.rust-toolchain }} llvm-cov nextest --features ci --all-targets --no-fail-fast --lcov --output-path lcov.info
 
+      - name: Benches should at least build
+        run: cargo +${{ matrix.rust-toolchain }} bench --features bench --no-run
+
```

Comment:
Ditto?

Reply:
Yeah, I'm trying to get benches built, in anticipation of one of the benchmarks landing.  Though I can move that to the other PR.

Reply:
No, leave it in. I just wanted to check that it's deliberate.